lib/game.rb

```diff
@@ -52,37 +50,35 @@ def self.average_goals_by_season
   def self.average_goals_by(hoa_team)
     divide_hash_values(:goals, :games_played, games_goals_by(hoa_team))
   end
-#deliverable
```

Comment:
it seemed ok at this point to remove these. If that's not ok, I'll put them back.

Reply:
Agreed I think it looks better without them